src/containerapp/azext_containerapp/_params.py

```diff
@@ -370,6 +370,11 @@ def load_arguments(self, _):
         c.argument('max_replicas', type=int, help="Maximum number of replicas to run for the Java component.")
         c.argument('route_yaml', options_list=['--route-yaml', '--yaml'], help="Path to a .yaml file with the configuration of a Spring Cloud Gateway route. For an example, see https://aka.ms/gateway-for-spring-routes-yaml")
 
+    with self.argument_context('containerapp env maintenance-config') as c:
+        c.argument('weekday', options_list=['--weekday', '-w'], arg_type=get_enum_type(["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday" ]), help="The weekday to schedule the maintenance configuration.")
+        c.argument('start_hour_utc', options_list=['--start-hour-utc', '-s'], help="The hour to start the maintenance configuration.")
+        c.argument('duration', options_list=['--duration', '-d'], help="The duration in hours of the maintenance configuration.")
```

Comment:
Since, you have it in validation
```suggestion
        c.argument('duration', options_list=['--duration', '-d'], help="The duration in hours of the maintenance configuration. Minimum value: 8")
```